doc/rst/usingchapel/editors/vscode.rst

```diff
@@ -0,0 +1,30 @@
+.. _vscode:
+
+VSCode
+------
+
+.. _vscode-syntax:
+
+Syntax Highlighting
+~~~~~~~~~~~~~~~~~~~
+
+Syntax highlighting is provided by the ``Chapel Language`` extension.
+To install the extension, visit the
+`Visual Studio Code marketplace <https://marketplace.visualstudio.com/items?itemName=chpl-hpe.chapel-vscode>`_.
+
+.. _vscode-lsp:
+
+Language Server Support
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``Chapel Language`` extension provides builtin support for the :ref:`Chapel language server <readme-chpl-language-server>` and :ref:`readme-chplcheck`. If ``chpl-language-server`` and ``chplcheck`` are in your PATH, the extension will automatically use it. If not, you can either specify you ``CHPL_HOME`` in the extension settings or specify the path to the executables.
+
+Specifying ``CHPL_HOME``:
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Open the settings for the Chapel extension and set ``chapel.CHPL_HOME`` to the path of your Chapel ``CHPL_HOME``.
```

Comment:
Mention that it tries to infer this (it does, doesn't it?)

Reply:
Yes, the above paragraph has this

> If ``chpl-language-server`` and ``chplcheck`` are in your PATH, the extension will automatically use it. If not, you can either specify you ``CHPL_HOME`` in the extension settings or specify the path to the executables.

This sentence is what to do if infering fails